spark/src/main/kotlin/com/adevinta/spark/InternalSparkApi.kt

```diff
@@ -25,17 +25,16 @@ package com.adevinta.spark
 /**
  * Marks declarations that are **internal** in Spark API, which means that should not be used outside of
  * `com.adevinta.spark`, because their signatures and semantics will change between future releases without
- * any warnings and without providing any migration aids.
+ * any warnings and without providing any migration guide.
  */
 @Retention(value = AnnotationRetention.BINARY)
 @Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION, AnnotationTarget.TYPEALIAS, AnnotationTarget.PROPERTY)
 @RequiresOptIn(
     level = RequiresOptIn.Level.ERROR,
-    message = "This is an internal com.adevinta.spark API that " +
-            "should not be used from outside of com.adevinta.spark. No compatibility guarantees are " +
-            "provided. It is recommended ask your Product Owner or your designer to modify your interface to respect " +
-            "Spark design system. As a last resort you can report your use-case of internal API to" +
-            " com.adevinta.spark owners, so stable API could be provided instead in a next version " +
-            "of Spark",
+    message = """This is an internal com.adevinta.spark API that should not be used from outside of com.adevinta.spark. 
+        |
+        |No compatibility guarantees are provided.
+        |
+        |It is recommended to create a feature request, so the Spark team can see with you if the existing API needs to be more open in a next version of Spark""",
```

Comment:
Raw string does not necessarily need delimiter, and is easier to read like this:

```kotlin
message = """
    This is an internal com.adevinta.spark API that should not be used from outside of com.adevinta.spark.
    No compatibility guarantees are provided.
    It is recommended to create a feature request, so the Spark team can see with you if the existing API needs to be more open in a next version of Spark.
    """.trimIndent(),
```

Reply:
We can't use `trimIndent()` since it's in a annotation so it need to use a compile-time constant :(

Reply:
Ah! ok, but then using `|` will remain in the string?

Reply:
Fixed in [45f4d18](https://github.com/adevinta/spark-android/pull/76/commits/45f4d181d64ca9be0924433b3bd3f4ce55cf65d1)